src/friend_oauth2/workflow.clj

```diff
@@ -28,19 +28,16 @@
 (defn- request-token
   "POSTs request to OAauth2 provider for authorization token."
   [{:keys [uri-config access-token-parsefn]} code]
-  (let [access-token-uri (:access-token-uri uri-config)
-        query-map        (-> (util/replace-authz-code access-token-uri code)
-                             (assoc :grant_type "authorization_code"))
-        token-parse-fn   (or access-token-parsefn util/extract-access-token)]
+  (let [access-token-uri     (:access-token-uri uri-config)
```

Comment:
Heads-up: indenting like this is a pain to maintain, so we don't do it. I'll merge this anyway, since it's been a while, but expect the whitespace here to change in the future, back to what it was.